js/views/pageNavVw.js

```diff
@@ -193,8 +228,17 @@ module.exports = Backbone.View.extend({
     loadTemplate('./js/templates/pageNav.html', function(loadedTemplate) {
       self.$el.html(loadedTemplate(self.model.toJSON()));
       if(localStorage.getItem("onboardingComplete") != "true") {
-        self.$el.find('.js-homeModal').removeClass("hide");
+        var modal = self.$el.find('.js-homeModal');
+        modal.removeClass("hide");
         $('#obContainer').addClass("blur");
+        modal.attr("tabIndex", "0");
+        document.addEventListener('focus', function( ev ) {
+          if ( !modal.hasClass("hide") && !$.contains( modal[0], ev.target ) ) {
+            ev.stopPropagation();
+            modal.focus();
+          }
+        }, true);
```

Comment:
It would probably be better to add this listener in app.js, and apply it to all elements with the modal class. But not required for this PR.